book/bundles.rst

```diff
@@ -0,0 +1,175 @@
+.. index::
+   single: Bundles
+
+.. _page-creation-bundles:
+
+The Bundle System
+-----------------
+
+A bundle is similar to a plugin in other software, but even better. The key
+difference is that *everything* is a bundle in Symfony, including both the
+core framework functionality and the code written for your application.
+Bundles are first-class citizens in Symfony. This gives you the flexibility
+to use pre-built features packaged in `third-party bundles`_ or to distribute
+your own bundles. It makes it easy to pick and choose which features to enable
+in your application and to optimize them the way you want.
+
+.. note::
+
+   While you'll learn the basics here, an entire cookbook entry is devoted
+   to the organization and best practices of :doc:`bundles </cookbook/bundles/best_practices>`.
```

Comment:
Ignore this entry for now - it's copied from the original. It needs to be updated for sure, but I'm saving that for another PR (but I will fix the title underlines - you're right about that)